lib/BaseController.js

```diff
@@ -85,16 +85,8 @@ module.exports = class BaseController {
         throw Error('Unrecognized object received from watch event');
       }
 
-      // TODO: https://github.com/razee-io/razeedeploy-core/issues/189
-      let impersonateUser = objectPath.get(this._data, 'object.spec.clusterAuth.impersonateUser', 'false');
-      if (impersonateUser !== 'razeedeploy') {
-        if (impersonateUser !== 'false' && this.namespace === 'razeedeploy') {
-          this._kubeResourceMeta.addHeader('Impersonate-User', impersonateUser);
-        } else {
-          impersonateUser = 'razeedeploy';
-          return await this.patchSelf({ 'spec': { 'clusterAuth': { 'impersonateUser': impersonateUser } } });
-        }
-      }
+      let impersonateUser = objectPath.get(this._data, 'object.spec.clusterAuth.impersonateUser', 'razeedeploy');
+      this._kubeResourceMeta.addHeader('Impersonate-User', impersonateUser);
```

Comment:
if impersonateUser is razeedeploy, we **shouldn't** add the header.

Reply:
addressed.